tutorials/hoon/arms-and-cores.md

```diff
@@ -99,7 +99,7 @@ First let's use the dojo to bind the face `a` to the value `12`, and the face `b
 
 Now use the following multi-line expression to bind `c` to a core.  As before, feel free to cut and paste the following expression into the dojo (starting at `=c`):
 
-```hoon
```

Comment:
-1 for removing these. It's good metadata to have around.

Reply:
They're dojo prompts though. We don't use that tag for dojo elsewhere in the docs.

Reply:
I definitely think we should have another tag for dojo, but that's a separate issue.

Reply:
When I was tagging all the code blocks during the urbit.org write I sometimes slipped `hoon` in if the code block was predominantly Hoon (not Dojo specific syntax).